anx/provider/loadbalancer/address/address.go

```diff
@@ -151,12 +150,12 @@ func (m *mgr) prefixes(ctx context.Context) ([]*prefix, error) {
 		ret = append(ret, m.fixedPrefixes...)
 	}
 
-	if m.prefixAutodiscover != nil {
+	if m.autoDiscoveryName != nil {
 		ctx, cancel := context.WithCancel(ctx)
 		defer cancel()
 
 		var oc types.ObjectChannel
-		err := m.api.List(ctx, &corev1.Resource{Tags: []string{*m.prefixAutodiscover}}, api.ObjectChannel(&oc), api.FullObjects(true))
+		err := m.api.List(ctx, &corev1.Resource{Tags: []string{fmt.Sprintf("kubernetes-lb-prefix-%s", *m.autoDiscoveryName)}}, api.ObjectChannel(&oc), api.FullObjects(true))
```

Comment:
code style nitpick: could we have this `Sprintf` (and the one below) on their extra line? Makes it a bit more readable

Reply:
Sure, will do.

Reply:
Done.